roxie/udplib/udpsha.cpp

```diff
@@ -171,29 +164,48 @@ DataBuffer *queue_t::pop()
 }
 
 
-bool queue_t::removeData(const void *key, PKT_CMP_FUN pkCmpFn)
+unsigned queue_t::removeData(const void *key, PKT_CMP_FUN pkCmpFn)
 {
-    bool ret = false;
-    CriticalBlock b(c_region);
-    if (active_buffers) 
+    unsigned ret = 0;
```

Comment:
picky: Not the most descriptive variable name.

Reply:
renamed and repushed